### PR TITLE
fix(apps: auth: db: commands): make --admin boolean

### PR DESCRIPTION
**Pull request by actionless**
resolves https://github.com/superdesk/superdesk/pull/1729

(argparse should do converting job for us)

### CODE REVIEW

**Comment by petrjasek** [comment]
:+1:

**Comment by mdhaman** [comment]
:+1:

File: apps/auth/db/commands.py [hunk 1]
```diff
@@ -27,13 +27,12 @@ class CreateUserCommand(superdesk.Command):
         superdesk.Option('--username', '-u', dest='username', required=True),
         superdesk.Option('--password', '-p', dest='password', required=True),
         superdesk.Option('--email', '-e', dest='email', required=True),
-        superdesk.Option('--admin', '-a', dest='admin', required=False),
+        superdesk.Option('--admin', '-a', dest='admin', required=False, default=False),
```
Comment (petrjasek) [inline apps/auth/db/commands.py]: ideally it should have also `action='store_true'` so that it can be used without any value, like `--admin`

Comment (actionless) [inline apps/auth/db/commands.py]: isn't it behave like that for bools by default?

Comment (petrjasek) [inline apps/auth/db/commands.py]: don't think so

Comment (actionless) [inline apps/auth/db/commands.py]: https://github.com/smurfix/flask-script/blame/master/docs/index.rst#L249-L269

Comment (petrjasek) [inline apps/auth/db/commands.py]: that probably works if you don't use `Option` and it does config automatically
btw `dest` should be `is_admin`

Comment (actionless) [inline apps/auth/db/commands.py]: under the hood it creates `Option` instances for them anyway

renamed it back to `admin` to avoid any confusion

Comment (petrjasek) [inline apps/auth/db/commands.py]: yep it does and it does some detection, but when you set it explicitly it doesn't do it.
I've tested it - if you run it with just `--admin` you get `manage.py users:create: error: argument --admin/-a: expected one argument`

Comment (actionless) [inline apps/auth/db/commands.py]: weird, after the sleep i will try to figure that out

**Comment by actionless** [comment]
so adding `, action='store_true'` as Petr suggested did help, i tested it works as expected now

**Comment by petrjasek** [comment]
it will require some updates in scripts because not it will fail with `--admin=true`

**Comment by actionless** [comment]
prepared: https://github.com/superdesk/superdesk/pull/1742

**Comment by petrjasek** [comment]
:+1:

**Comment by actionless** [comment]
(will get back to it after the server migration which will be on the weekend)

**Pull request merged**